hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Should be fairly straight forward fix (depending on how crazy you want to get).  The simple solution would be get a list of the years in the current copyright header, get a list of years in the git history and take the union of the two lists and that's this list of years that gets updated into the header.  There are other potential scenarios that could cause problems, for example what do you do if the current file has a year that's beyond the first year in the git log but the git log doesn't show an update for that year.  Do you keep that year it or throw it out?  Thinks could get overly complicated quick if we're not careful.